src/pages/AdminPages/Transactions/PendingTransactions.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import BaseModal from '../../../components/Modal/BaseModal'
+import { Table } from '../../../components/Table'
+import { setSelectedAlterContent, sendTransactionResponse } from '../../../store/reducers/adminQuery'
+import { Form, SubmitButton, TextareaField } from '../../../components/Form'
+import { disableButton } from '../../../store/reducers/buttonState'
+import * as Yup from 'yup'
+import { thousands_separators, timeAgo, TIME_ZONE } from '../../../utils/helperFunctions'
+
+
+const validationSchema = Yup.object().shape({
+    reason: Yup.string().required().min(3).label("Declination Reason")
+})
+
+
+const PendingTransactions = ({ 
+    pendingTransactions,
+    processedTransactions,
+    selectedAlterContent,
+    buttonDisable,
+    header,
+    setContent
+}) => {
+
+
+    const [showResponse, setShowResponse] = useState(false)
+
+    const dispatch = useDispatch()
+
+    const closeShowResponse = () => {
+        dispatch(setSelectedAlterContent({}))
+        setShowResponse(false)
+    }
+
+
+    const processTransaction = ({ reason, status }) => {
+
+        const transactionResponse = {
+            processedTransactions,
+            pendingTransactions,
```

Comment:
Why do we have to pass these two to process one transction

Reply:
Bcos I'm holding each array for updating after successful transaction... you can look through the `handler` section.